server/app/models/list.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: lists
+#
+#  id             :integer          not null, primary key
+#  description    :text
+#  items_quantity :integer          default(0), not null
```

Comment:
The automatic name would be `list_items_count`. But I prefer to change to `items_count`, since I define the relationship like `items` instead of `list_items`, what do you think?